cmd/acra-webconfig/acra-webconfig.go

```diff
@@ -389,7 +389,6 @@ func main() {
 	host = flag.String("incoming_connection_host", cmd.DefaultWebConfigHost, "Host for AcraWebconfig HTTP endpoint")
 	port = flag.Int("incoming_connection_port", cmd.DefaultWebConfigPort, "Port for AcraWebconfig HTTP endpoint")
 	loggingFormat := flag.String("logging_format", "plaintext", "Logging format: plaintext, json or CEF")
-	logging.CustomizeLogging(*loggingFormat, ServiceName)
```

Comment:
why this is removed? the next line is a log line, so if logger is not customized, the log line will be output in unknown/default format

Reply:
this line should be here

`logging.Customize().SetFormat(*loggingFormat).SetServiceName(ServiceName).Complete()`